src/animations/AnimationManager.js

```diff
@@ -238,7 +238,7 @@ var AnimationManager = new Class({
      * @property {string} [suffix=''] - [description]
      * @property {integer} [zeroPad=0] - [description]
      * @property {AnimationFrameConfig[]} [outputArray=[]] - [description]
-     * @property {boolean} [frames=false] - [description]
+     * @property {boolean|number[]} [frames=false] - [description]
```

Comment:
Please wrap the values: `{(boolean|integer[])}` (integer makes more sense as the type here, as a float is invalid)